content/directory/PLU MarCom.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "PLU MarCom"
+cover: "./content/blog/imgs/hamburgers.png"
+website: "https://www.plu.edu/marcom/"
+twit: ""
+inst: ""
+category: "P"
+tags:
+    - Tacoma
+    - small
+    - marketing
+    - advertising
+    - photograhy
+    - print
+    - ux/ui
+    - video
```

Comment:
let's add `-school` here as a tag. I don't think the other schools I tagged had as much detail. Would this entry replace the **plu.edu/artd** entry?  We may need to revisit the the others as well. https://seattlecreative.directory/tags/school, Looks like Seattle Central has a similar issue where there are multiple urls that kinda lead to the same place.